src/wazuh_modules/vulnerability_detector/wm_vuln_detector_db.h

```diff
@@ -228,8 +229,9 @@ static const char *vu_queries[] = {
     "SELECT ID FROM NVD_CVE_CONFIGURATION WHERE NVD_CVE_ID = ?;",
     // NVD REPORT
     "SELECT CVE_ID, CWE_ID, ASSIGNER, DESCRIPTION, VERSION, PUBLISHED, LAST_MODIFIED FROM NVD_CVE WHERE ID = ?;",
+    "SELECT ID, CWE_ID, ASSIGNER, DESCRIPTION, VERSION, PUBLISHED, LAST_MODIFIED FROM NVD_CVE WHERE CVE_ID = ?;",
     "SELECT CVE_ID FROM NVD_CVE WHERE ID = ?;",
-    "SELECT URL, REF_SOURCE FROM NVD_REFERENCE WHERE NVD_CVE_ID = ? LIMIT 1;",
+    "SELECT URL, REF_SOURCE FROM NVD_REFERENCE WHERE NVD_CVE_ID = ?;",
```

Comment:
Not sure why the query to get the NVD references was limited to 1 row. This was causing a big loss of reference information very important for the alerts.